integration-test/src/test/java/org/ehcache/integration/statistics/TierCalculationTest.java

```diff
@@ -328,7 +328,7 @@ public void testEviction() {
   @Test
   public void testExpiration() throws InterruptedException {
     cache.put(1, "a");
-    Thread.sleep(100);
+    Thread.sleep(300);
```

Comment:
huh? please inject a `TimeSource` that enables you to simulate time - that `sleep` needs to go